CONTRIBUTING.md

```diff
@@ -1,3 +1,5 @@
 # Contributing Guide
 
 Please see our [general guide for OSS lifecycle and practices.](https://github.com/honeycombio/home/blob/main/honeycomb-oss-lifecycle-and-practices.md)
+
+When modifying a chart you must increment the chart's version, following [semvar](https://semver.org/).
```

Comment:
```suggestion
When modifying a chart you must increment the chart's version, following [semver](https://semver.org/).
```

Reply:
Can we give a specific example here? Like "For example, if making a change to the Refinery chart, increment the `version` in the `Chart.yaml` file for that chart."